labels_new.md

```diff
@@ -0,0 +1,221 @@
+# Labels Proposal
+
+## Goals
+
+* synchronise labels between repos, especially priority labels
```

Comment:
Propose to take a look and try https://github.com/marketplace/actions/label-syncer

As `Sync labels on another repository` section says, it can specify a repository as the source of label config, then sync all labels with multiple repositories by this remote config.